Update Ubuntu-Install-Debians.rst

SuperKenVery opened the pull request:
Though `.bash` in the file name is quite explicit, hard-coding the `.bash` in this document isn't that good, I think. 

It was not until I used `Tab` to complete the command line that I found there is a `zsh` version, after getting errors sourcing the `bash` version with `zsh`.

SuperKenVery committed:
Update Ubuntu-Install-Debians.rst

Though `.bash` in the file name is quite explicit, hard-coding the `.bash` in this document isn't that good, I think. It was not until I used `Tab` to complete the command line that I found there is a `zsh` version, after getting errors sourcing the `bash` version with `zsh`.

fujitatomoya commented on source/Installation/Ubuntu-Install-Debians.rst:
```diff
@@ -77,6 +77,8 @@ Set up your environment by sourcing the following file.
 .. code-block:: bash
 
    source /opt/ros/{DISTRO}/setup.bash
+   #Replace ".bash" with your shell if you're not using bash
+   #Possible values are: setup.bash, setup.sh, setup.zsh
```
```suggestion
   # Replace ".bash" with your shell if you're not using bash
   # Possible values are: setup.bash, setup.sh, setup.zsh
   source /opt/ros/{DISTRO}/setup.bash
```

fujitatomoya commented on source/Installation/Ubuntu-Install-Debians.rst:
```diff
@@ -91,13 +93,17 @@ In one terminal, source the setup file and then run a C++ ``talker``\ :
 .. code-block:: bash
 
    source /opt/ros/{DISTRO}/setup.bash
+   #replace ".bash" with your shell if you're not using bash
+   #Possible values are: setup.bash, setup.sh, setup.zsh
```
```suggestion
   # Replace ".bash" with your shell if you're not using bash
   # Possible values are: setup.bash, setup.sh, setup.zsh
   source /opt/ros/{DISTRO}/setup.bash
```

fujitatomoya commented on source/Installation/Ubuntu-Install-Debians.rst:
```diff
@@ -91,13 +93,17 @@ In one terminal, source the setup file and then run a C++ ``talker``\ :
 
 .. code-block:: bash
 
    source /opt/ros/{DISTRO}/setup.bash
+   #replace ".bash" with your shell if you're not using bash
+   #Possible values are: setup.bash, setup.sh, setup.zsh
```
```suggestion
   # Replace ".bash" with your shell if you're not using bash
   # Possible values are: setup.bash, setup.sh, setup.zsh
   source /opt/ros/{DISTRO}/setup.bash
```

fujitatomoya reviewed (approved):
lgtm with minor nitpicks

SuperKenVery committed:
Update source/Installation/Ubuntu-Install-Debians.rst

Co-authored-by: Tomoya Fujita <Tomoya.Fujita@sony.com>

SuperKenVery committed:
Update source/Installation/Ubuntu-Install-Debians.rst

Co-authored-by: Tomoya Fujita <Tomoya.Fujita@sony.com>

SuperKenVery committed:
Update source/Installation/Ubuntu-Install-Debians.rst

Co-authored-by: Tomoya Fujita <Tomoya.Fujita@sony.com>

clalancette reviewed (commented):
So the issue with this is that we have documentation that says to source `setup.bash` in about 150 places in the documentation.  Changing it in just this one place seems weird.  At the very least, we should change it in the rest of the Installation instructions, but personally I'd prefer if we didn't do this at all as we'd need to repeat ourselves a lot.

fujitatomoya commented:
let's not repeat this note everywhere. IMO, leaving note in the 1st place would be okay. Or i think this is just a `nice-to-have`, so `won't fix` would be fine too. @SuperKenVery what would you think?

SuperKenVery commented:
I think leaving note in the 1st place would be good.

clalancette committed:
Add to all of the installation pages.

Add the language to all of the installation pages, and also
to the first of the tutorials.  This seems to be the common
entry points for "new" users, so let's have it in all of these
places.

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

clalancette commented:
After discussing it with others, we came to the agreement that we should have this.  But I do think we should have it all of the reasonable "entry" points that people have with the documentation, so what I've done in the latest commits here is to add it to all of the Installation pages (where it makes sense), and also the very first Beginner tutorial.  If someone is just starting out, those are the most likely places they'll be looking.  With that, I'm happy with this so I'll approve and backport.

clalancette commented:
@Mergifyio backport rolling galactic foxy
